modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java

```diff
@@ -490,6 +535,8 @@ public Assignment getAssignment(User.ID userID, Application.Name applicationName
                     }
 
                     tempResult.put("status", assignment.getStatus());
+                    if(LOGGER.isDebugEnabled()) LOGGER.debug("tempResult: "+tempResult);
```

Comment:
Changed to use tempResult={} instead of string concatenation... but keeping idDebugEnabled as having isDebugEnabled avoids a string concatenation in the case if logging LEVEL is not debug..